app/lib/check_pusher.rb

```diff
@@ -7,88 +7,20 @@ def pusher_channel
     'check-channel-' + Digest::MD5.hexdigest(self.class_name + ':' + self.id.to_s + ':pusher_channel')
   end
 
-  def notify_pusher_channel
-    CheckPusher::Worker.perform_async(['check-api-global-channel'], 'update', { pusherChannels: [self.pusher_channel], pusherEvent: 'media_updated' }.to_json, self.class.actor_session_id)
-  end
-
   module ClassMethods
-    def pusher_options
-      @pusher_options
-    end
-
     def send_to_pusher(channels, event, data, actor_session_id)
       ::Pusher.trigger(channels, event, { message: data, actor_session_id: actor_session_id }) unless CheckConfig.get('pusher_key').blank?
     end
 
-    def pusher_options=(options)
-      @pusher_options = options
-    end
-
-    def notifies_pusher(options = {})
-      events = [options[:on]].flatten
-
-      pusher_options = self.pusher_options || {}
-
-      events.each do |event|
-        send("after_#{event}", ->(_obj) { notify_pusher(event) })
-        pusher_options[event] = options
-      end
-
-      self.pusher_options = pusher_options
-
-      send :include, InstanceMethods
-    end
-
     def actor_session_id
       RequestStore[:actor_session_id] ||= RequestStore[:request].blank? ? '' : RequestStore[:request].headers['X-Check-Client'].to_s
     end
   end
 
   module InstanceMethods
-    def sent_to_pusher
-      @sent_to_pusher
-    end
-
-    def sent_to_pusher=(bool)
-      @sent_to_pusher = bool
-    end
-
-    def parse_pusher_options(action)
-      options = self.class.pusher_options[action]
-      return if options.has_key?(:if) && !options[:if].call(self)
-
-      event = options[:event].is_a?(String) ? options[:event] : options[:event].call(self)
-      targets = options[:targets].call(self)
-      data = options[:data].call(self)
-
-      [event, targets, data]
-    end
-
     def actor_session_id
       self.class.actor_session_id
```

Comment:
@melsawy this line is uncovered, please check if this method can be safely removed.

Reply:
Removed